### DISCUSSION TITLE
Setting custom icons?

### DISCUSSION THREAD

**Discussion by probablySophie** [nvim-tree/nvim-tree.lua#1738] [Q&A] [answered]
This is probably a dumb question, but I can't figure out how to set a custom icon for a file type/language in the file tree.

Can anyone explain how to do it?

Thank you

**Comment by alex-courtis** [comment, marked as answer]
Icons are handled by nvim-wed-devicons.

You can override the icon for a specific type: https://github.com/nvim-tree/nvim-web-devicons#set-an-icon
